core/src/main/java/com/linecorp/armeria/common/metric/CertificateMetrics.java

```diff
@@ -73,7 +73,7 @@ public void bindTo(MeterRegistry registry) {
                          .description(
                                  "1 if TLS certificate is in validity period, 0 if certificate is not in " +
                                  "validity period")
-                         .tags("common.name", commonName)
+                         .tags("hostname", hostname)
```

Comment:
This is a breaking. Could you update the description so that this breaking change can be included in the note?

Reply:
> Breaking) Rename common.name tag to hostname tag in certificate metrics.

I'd already added one for that. Do you think it is not enough?

Reply:
Oops, forget about it. 😓